### ISSUE TITLE
Desktop just freezes/dies and eventually stops starting when changing monitor/resolutions

### ISSUE THREAD

**Issue by maxandersen** [devhubapp/devhub#209]
I can no longer start the desktop app. 

When it start it just freezes.

This seem to be culmination of the app freezing when changing monitor/use sidecar/switching resolutions - on every of those the app freezes and last I had it working on startup it stated something like "the app crashed, do you want to reopen the same windows or not" ...I clicked not by accident and now no windows shows up and the app is frozen.

any pointers on how to get the app working again would be great :)

**Comment by brunolemos** [comment]
Weird. Have you tried reinstalling?

**Comment by brunolemos** [comment]
I'm not able to reproduce. I tried changing the macOS resolution on both `desktop` and `menubar` modes. Do you have any suggestion of what else I could try to reproduce? Also please send a screenshot of the issue and the stacktrace if any.

**Comment by maxandersen** [comment]
yes, now tried uninstalling and reinstalling. same reaction - app just freezes, never starts.

I've removed the app from /Applications and ~/Library/Application Support and then downloaded - its just stuck with only option on the dock icon to click "Force Quit..."

about reproducing - What I've been doing is just connecting/disconnecting external monitors (real ones and ipad sidecar).

**Comment by maxandersen** [comment]
so i've now nuked and removed everything multiple times and nothing changes. the app simply never starts.

one thing I do notice is that when launching devhub it tries to access all my network drives old as well new which aren't available right now when i'm travelling.

could there be some logic/config that are triggering it to pull all network drives that makes it get stuck ?

**Comment by brunolemos** [comment]
I really don't know what could be causing this. I need to see the crash log.
I'm not sure which folder it is, maybe `~/Library/Logs/DiagnosticReports`?

I also will add some crash reporting tool to the electron part of the app and release a new version to investigate this.

**Comment by maxandersen** [comment]
It doesn't crash - it just freezes. I have to force kill it. If you let me know what to download, enable of logs etc. i'm happy to run it. 

Mind you - I strongly believe its something happening initially in the bootup - when I start devapp I get multiple dialog boxes stating "there was a problem connecting to the server <name of my NAS at home>" as if its going through all old network volumes of some sort...and if I keep closing those dialogs its as if the app unfreezes but shortly after a new dialog pops up with exact same message.... eventually it stops but app still frozen.

for  now i'm using the ipad app to "survive" :)

**Comment by brunolemos** [comment]
> I strongly believe its something happening initially in the bootup

Have you tried the solutions listed below?

https://apple.stackexchange.com/questions/98177/endlessly-repeating-there-was-a-problem-connecting-error/131788

**Comment by maxandersen** [comment]
None of those solutions works.

Now that I'm back home (where the remote volume is again available) devhub started to run again.

so definitely some kind of filescanning/lookup thing going on that was causing it to freeze.

just wondering if it will fail again when I travel the next time ;/

**Comment by almaashy-99** [comment]
Mohammed

**Comment by Vishv07** [comment]
does this app still maintaining by someone? or it's up for a grabs

**Comment by brunolemos** [comment]
@Vishv07 it's open to receive pull requests and then I'll release a new version with the fix
